### DISCUSSION TITLE
@types/node support for `cluster.isWorker`

### DISCUSSION THREAD

**Discussion by evantahler** [DefinitelyTyped/DefinitelyTyped#54470] [Issues with a @types package]
@types/node at v16.3.1 doesn't support `cluster.isWorkers` - https://nodejs.org/api/cluster.html#cluster_cluster_isworker

```
src/classes/process/id.ts:25:17 - error TS2551: Property 'isWorker' does not exist on type 'typeof import("cluster")'. Did you mean 'Worker'?

25     if (cluster.isWorker) {
                   ~~~~~~~~
```

A PR with an example can be seen here https://github.com/actionhero/actionhero/pull/1883

**Comment by ronjouch** [comment]
Confirmed, `cluster` 16.x types are unusable:

- DT types want you to use a default export...
- ... but the module itself does *not* export as default module!

See [TypeScript playground](https://www.typescriptlang.org/play?esModuleInterop=false&declaration=false&experimentalDecorators=false&emitDecoratorMetadata=false&target=8&jsx=0&module=1&ssl=1&ssc=1&pln=2&pc=1#code/PTAEBUAsEsGdTqAhqAxgewA4E8C0mlYAXFdAM1ABFxhUAbAV2IFMAnUI7TZ+JI0SESKZYALhABzaEUgMARgDoMAW2CVmZaADtpzOtnBdmAEzUbtu-Ye6m5ddHODLCRNsE7dYwLemPNajCysCsYKRLAAxAAyAIwADLhRAMwA7ABQIBzooH7K6FrErHzMAugA7hyQJazMKsrMWn7GoMwAHkjKmHQlxug8WgDk-GT6oGXSkBAAyqBM2hJo6Mp5WgBWsAoZYFU1oAAUSI2gcnrlAJQANMhkruxIxsbzlSUARABUyPAvoGTo7D5aXB+MhIBh0fjQTp-cIXLagACSAnuWVAdGKdweoAAAuFcG1uKgiHjWKw-iiJMx+OAZkRsipMNBuqBRLgwCy2ay0pDMNDQB9CGhArcfqTlKABvQmLcBgBuLlQ1j8fnwQTCEVLcWqzCy+U8xWgADeaEwTFAAF91WKBuhYDrubzlaBMKTUDx4GRReLnehXbBbXK0hgCvwtAxlABhAAKAFV4ABeY1MPZnBTdLQSGQBzI42B41oEolsUnsdlcih7SVBBRwSOsSFIVjYM6GtKgRYFdDdVPoCR7AAGtfrjdAABIDd7fRsGcYLYhWAwtDp032zgG25kAGJ-ADWYx3bA2rZ+ZL23QhoATcRlCFAAB5QKGIzHYNfoABqN-Ng1H9dgHN5gtiWLZlOTbNtK1uBRflYbdkzlNszTSI9s1xfFakLEkyVLcChTYBR8j2AY2mkAYrj2Mp91YK4MD8K5YGgCQtCQOhmzjAA+FswPbWBO2Ybtez7CiYLYUcDSE7c8InN0FGnC1HhMFd4PNVc0gtPRYBKb9f1AAB1Sj4FQQ5QFgSAGxKQ5sAgKN2y0dDoHyZCwHhLRKkQQyNIQCFeBcgAJcBwEjYy2AANzYI8tSUGpiimEK2D2PYagARyuGpYFYjitLA1KFDKOtXB85h7j2AAmOI4hUrjsoaYxCKqOh7D3Vg6GMAAdQYKuU1M4FcLQ9gADjK8q1243j+P7PThPYMcpL9GToBnYySEVBSVLNIA) and [StackOverflow: How to use Node.js `cluster` module with DT @types/node@16.x and TypeScript/commonjs?](https://stackoverflow.com/questions/72548137/how-to-use-node-js-cluster-module-with-dt-types-node16-x-and-typescript-comm).

Past maintainers @SimonSchick @sandersn @y-i @jablko, can you halp?
